apps/static/src/assets/mirror-network.json

```diff
@@ -0,0 +1,3 @@
+{
+  "hosts": ["https://api.n04.d.vega.xyz/graphql"]
```

Comment:
Is this correct? Based on file names, I'd expect this file to contain "https://api.n00.mainnet-mirror.vega.xyz/graphql" and `apps/static/src/assets/devnet-network.json` to contain "https://api.n04.d.vega.xyz/graphql"

Reply:
thanks!